### ISSUE TITLE
Accessibility Issues on Install Screen

### ISSUE THREAD

**Issue by RachBreeze** [umbraco/UmbracoDocs#4512]
v11 Focus state is missing off the subscribeToNewsLetter checkbox and consent for telemetary data says 2.0 in screen reader

**Comment by sofietoft** [comment]
Thanks for raising issue @RachBreeze ! 💪 

As I understand, something is missing in Umbraco 11 - is that something missing from the docs or something in the CMS it self?
Would be great if you could add some more details to the issue here, as I'm not entire sure I understand how we can help resolve it at the moment 🙏

**Comment by RachBreeze** [comment]
Heya
Sorry @sofietoft. I did miss a key detail there! It's in v11 when you go to the install screen. When you tab to subscribeToNewsLetter checkbox the check box isnt showing a focused state so it's not clear it has been tabbed to.

And again in the install screen when you use a screen reader the screen reader is showing 2.0 for telemetary data rather than the displayed text.

**Comment by sofietoft** [comment]
Aah, I see 😄 Thanks for explaining!

I think this is actually a better fit on the Umbraco-CMS repo, as it's not really a docs-related issue, right? 🤔

**Comment by RachBreeze** [comment]
Erm yes you're right @sofietoft. I'll get my coat now. Sorry
